### ISSUE TITLE
Style/MultilineOperationIndentation doesn't catch lines ending in `\`

### ISSUE THREAD

**Issue by mockdeep** [rubocop/rubocop#1473]
When I wrap strings with `\` the `MultilineOperationIndentation` style check doesn't flag them when they are misindented. The following string passes checks:

``` ruby
      flash[:error] = 'Here is a string ' \
                      'That spans' <<
        'multiple lines'
```

Side note, it also allows `<<` for multi-line strings. I'll add another issue for that.
